package.json

```diff
@@ -98,7 +99,6 @@
     "ignore-loader": "^0.1.2",
     "jest": "^26.1.0",
     "lint-staged": "^10.2.11",
-    "node-fetch": "^2.6.0",
```

Comment:
Does this change produce any updated in `yarn.lock`?

Reply:
.lock file doesn't track the origin of the dependency